test/common/cro_cr_success.go

```diff
@@ -0,0 +1,128 @@
+package common
+
+import (
+	goctx "context"
+	"encoding/json"
+	"fmt"
+	crov1 "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1"
+	croTypes "github.com/integr8ly/cloud-resource-operator/pkg/apis/integreatly/v1alpha1/types"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/resources/constants"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+)
+
+const (
+	requestNameSpace  = "redhat-rhmi-operator"
+	requestUrl        = "/apis/integreatly.org/v1alpha1"
+	openShiftProvider = "openshift"
+	externalProvider  = "aws"
+)
+
+func TestCROPostGresSuccessfulState(t *testing.T, ctx *TestingContext) {
+	postGresToCheck := []string{
```

Comment:
We are currently using three different styles to list resources to check or loop through in a test:

A function: https://github.com/integr8ly/integreatly-operator/blob/ca9144506bb8108423a1a800c1a3670d6d8207cc/test/common/deployment_types.go#L98

A constant: https://github.com/integr8ly/integreatly-operator/blob/ca9144506bb8108423a1a800c1a3670d6d8207cc/test/common/alerts_exist.go#L51

I would personally prefer the constant, but until the same style is used in all tests for me it's fine, wdyt?

We can also solve this in a follow-up task

Reply:
Yeah, I'm good with using them as constants